routing/graph.go

```diff
@@ -68,7 +60,9 @@ func (g *dbRoutingTx) forEachNodeChannel(nodePub route.Vertex,
 		return cb(info, p1, p2)
 	}
 
-	return g.graph.ForEachNodeChannel(g.tx, nodePub[:], txCb)
+	return kvdb.View(g.graph.Database(), func(tx kvdb.RTx) error {
```

Comment:
Okay, I think I drop the commit for the moment since we have identified the actual culprit that breaks the itests. We'll need to address the path finding later on anyway.